### PR TITLE
need to make development environment same. 

### LINKED ISSUE #137: need to make development environment same.
**Eenvironment(local)**:
npm: 6.14.14
node: 14.17.5

**Description**:
When I install dependencies using "npm install" command, the "package-lock.json" file is updated unlike the existing one.

image

**Why**:
As we can check the lockFileVersion is different on the above image, I guess this problem is from the development environment is different on person.

**Related issue**:
issue #136

### PR DESCRIPTION
**Pull request by starkensin**
#### What are the relevant tickets?

Fixes #137 

### Checklist
- [X] Added relevant tests or not required
- [X] Didn't break anything

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,14 +3,7 @@
...
 ## Developing Yorkie CodePair
-
-To develop CodePair in a containerized environment:
-
-```
-docker-compose -f docker/docker-compose.dev.yml up --build -d
-```
-
-Or to run the CodePair in the local environment:
+For building Yorkie CodePair, You'll first need `node.js` installed (node.js version 16+ and npm version 7.10+ are required).
```
Comment (hackerwins) [inline README.md:6]:
```suggestion
For building Yorkie CodePair, You'll first need Node.js installed(Node.js version 16+ and npm version 7.10+ are required).
```

**Review by hackerwins — APPROVED** [review]
LGTM. Thanks for your contribution.

**Comment by hackerwins** [comment]
If one of @mojosoeun, @ppeeou, and @blurfx approves, I think it's time to merge them.

**Review by blurfx — APPROVED** [review]
LGTM 👍

**Pull request merged**